lib/things.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, url):
     def create(self, thing, token):
         '''Creates thing entity in the database'''
         mf_resp = response.Response()
-        http_resp = requests.post(self.url + "/things", json=thing, headers={"Authorization": token})
+        http_resp = requests.post(self.url + "/things/bulk", json=thing, headers={"Authorization": token})
```

Comment:
AFAIK thing can be create:
1. As a single thing (using `/things` URL)
2. Multiple things at the same time, in bulk (using `/things/bulk`)

So instead of changing this funcition, it would be better to make new (an additional) `create_bulk()` function.

Reply:
From the docs `create thing` calls `/things/bulk` as indicated [here](https://docs.mainflux.io/api/#create-thing) which is the same endpoint with `create things`

Reply:
As discussed, both options should be supported (create single and bulk), and documentation needs to be corrected.

Reply:
They are already supported in the existing sdk

Reply:
What do you mean? As I can see here, you changed `create()` function, and what I suggested is not to change `create()`, but to add `create_bulk()` function.